modules/salt/pages/large-scale.adoc

```diff
@@ -174,3 +174,15 @@ If significant non-transient swap usage is detected, you can increase the availa
 
 You can also consider tuning the Server to consume less memory.
 For more information on tuning, see xref:salt:large-scale-tuning.adoc[].
+
+=== AES Key rotation
+
+Communication from the Salt Master to all Salt minions is encrypted with a single AES key. This key is rotated whenever:
+ - the salt-master process is restarted, or
+ - any Salt minion key is deleted (eg. a client is deleted from {productname})
```

Comment:
```suggestion
 * The ``salt-master`` process is restarted, or
 * Any client key is deleted (for example, when a client is deleted from {productname})
```

Reply:
I am taking your suggested edit but will stick to "minion key" here, as there are multiple keys involved for any client and here we refer to a very specific one (sorry, crypto is picky!)